paper-ripple-behavior.html

```diff
@@ -66,10 +66,10 @@
     /**
      * Ensures this element contains a ripple effect. For startup efficiency
      * the ripple effect is dynamically on demand when needed.
-     * @param {!Event=} opt_triggeringEvent (optional) event that triggered the
+     * @param {!Event=} optTriggeringEvent (optional) event that triggered the
      * ripple.
      */
-    ensureRipple: function(opt_triggeringEvent) {
+    ensureRipple: function(optTriggeringEvent) {
```

Comment:
btw, the JavaScript style guide specifically asks for `opt_` here (this was not a mistake).

https://google.github.io/styleguide/javascriptguide.xml?showone=Naming#Naming